### PR TITLE
fix "load more" videos button

### LINKED ISSUE #60: Load More button acts unpredictable
When you hit `load more` a couple of times it should look like this at the end;

- load_more_bug2

But if you keep hitting the button, it looks like this;

- load_more_bug4

Need to fix duplicate videos and disable the `load more` button when there are no more videos to show

### PR DESCRIPTION
**Pull request by amgando**
- remove BaseButton bc CSS made too large a target for @click event
- update cache to discard duplicate content IDs
- update composable to detect that no content changed since last fetch

closes #60

### CODE REVIEW

**Comment by amgando** [comment]
Heads up, button behavior is still suboptimal

The last time you click, the button does nothing and then disappears. We can also just disable it.

This could be improved by predicting the total number of records and hiding / disabling before the last click

File: src/composables/cache.js [hunk 1]
```diff
@@ -74,6 +74,14 @@ class Cache {
     if (this.data[collection]) {
       if (collection === 'authors') {
         this.data[collection] = data;
+      } else if (/translations/.test(collection)) {
+        const newItems = [];
+        data.map(item => {
+          if (!this.find(collection, item.id)) {
+            newItems.push(item);
+          }
+        });
```
Comment (mehtaphysical) [inline src/composables/cache.js:83]:
```suggestion
        const newItems = data.filter(item => !this.find(collection, item.id)));
```

**Review by mehtaphysical — APPROVED** [review]
looks good. One little thing

**Review by hiba-machfej — APPROVED** [review]

File: src/composables/cache.js (hunk 1 above)
Comment (hiba-machfej) [reply src/composables/cache.js:83]: @amgando

Comment (amgando) [reply src/composables/cache.js:83]: yep, better, thank you

**Pull request merged**